# Request
uint16 mode1
uint16 mode2
---
# Response
bool success
